api/to_log.go

```diff
@@ -0,0 +1,195 @@
+// Copyright 2020 Author: Ruslan Bikchentaev. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package api
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ruslanBik4/dbEngine/dbEngine"
+	"github.com/ruslanBik4/logs"
+	"github.com/valyala/fasthttp"
+
+	"github.com/ruslanBik4/uppeople/auth"
+	"github.com/ruslanBik4/uppeople/db"
+)
+
+func HandleReturnLogsForCand(ctx *fasthttp.RequestCtx) (interface{}, error) {
+	DB, ok := ctx.UserValue("DB").(*dbEngine.DB)
```

Comment:
We have here & above some code that get DB from ctx & invoke some its method.
Its' must be move into  a function